src/vmm/src/lib.rs

```diff
@@ -512,10 +525,21 @@ mod tests {
         let temp_file = TempFile::new().unwrap();
         vmm_config.kernel_config.path = PathBuf::from(temp_file.as_path());
         let mut vmm = mock_vmm(vmm_config);
+        assert!(matches!(
+            vmm.load_kernel().unwrap_err(),
+            Error::KernelLoad(loader::Error::Bzimage(
+                loader::bzimage::Error::InvalidBzImage
+            ))
+        ));
 
+        // Test case: kernel path doesn't point to a file.
```

Comment:
It looks like an empty file would yield the same error. Do we need a directory?

Reply:
Nope, it's the same code path so we gain nothing on coverage, but it's an edge case nonetheless.